introduction.md

```diff
@@ -0,0 +1 @@
+My name is Martynas and it is the first time I am using Git Shell
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.